docs/Modern-Caching.md

```diff
@@ -0,0 +1,70 @@
+---
+id: relay-caching
+title: Caching
+---
+
+Relay modern makes no assumptions about what to cache and will garbage collect any data that is no longer referenced.
+
+You have to implement your own cache strategy. A simple solution is to use `RelayQueryResponseCache`(an in-memory cache) at the network layer:
+
+```javascript
+import { Environment, Network } from 'relay-runtime';
+import RelayQueryResponseCache from 'relay-runtime/lib/RelayQueryResponseCache';
+
+const oneMinute = 60 * 1000;
+const cache = new RelayQueryResponseCache({ size: 250, ttl: oneMinute });
+
+function fetchQuery(
+  operation,
+  variables,
+  cacheConfig,
+) {
+  const queryID = operation.text;
+  const isMutation = operation.query.operation === 'mutation';
+  const isQuery = operation.query.operation === 'query';
```

Comment:
In Relay 1.5.0 I had to change `operation.query.operation` to `operation.operationKind`. Not sure if and when this changed, though.

Reply:
You're absolutely right